test/modules/stackables/components/BottomSheet.test.js

```diff
@@ -57,4 +70,36 @@ describe('BottomSheet', () => {
 			expect(contentElement.innerText).toMatch(/FooBarBaz[\r\n]?/);
 		});
 	});
+
+	describe('responsive layout ', () => {
+
+		it('layouts for landscape and open Menu', async () => {
+			const element = await setup('FooBar', { mainMenu: { open: true }, media: { portrait: false } });
+			const contentElement = element.shadowRoot.querySelector('.bottom-sheet');
+
+			expect(contentElement.innerText).toContain('FooBar');
+			expect(element.shadowRoot.querySelectorAll(`[${TEST_ID_ATTRIBUTE_NAME}]`)).toHaveSize(1);
+			expect(element.shadowRoot.querySelectorAll('.bottom-sheet.is-open')).toHaveSize(1);
+		});
+
+		it('layouts for landscape and closed Menu', async () => {
+			const element = await setup('FooBar', { mainMenu: { open: false }, media: { portrait: true } });
+			const contentElement = element.shadowRoot.querySelector('.bottom-sheet');
+
+			expect(contentElement.innerText).toContain('FooBar');
+			expect(element.shadowRoot.querySelectorAll(`[${TEST_ID_ATTRIBUTE_NAME}]`)).toHaveSize(1);
+			expect(element.shadowRoot.querySelectorAll('.bottom-sheet.is-open')).toHaveSize(0);
+		});
+
+		it('layouts for portrait and open Menu', async () => {
+			const element = await setup('FooBar', { mainMenu: { open: true }, media: { portrait: true } });
+			const contentElement = element.shadowRoot.querySelector('.bottom-sheet');
+
+			expect(contentElement.innerText).toContain('FooBar');
+			expect(element.shadowRoot.querySelectorAll(`[${TEST_ID_ATTRIBUTE_NAME}]`)).toHaveSize(1);
+			expect(element.shadowRoot.querySelectorAll('.bottom-sheet.is-open')).toHaveSize(0);
+		});
```

Comment:
Is there no test for `portrait and closed Menu'` needed?

Should we also add tests for changing orientation or opening/closing the menu after initial rendering?

Reply:
done